### PR TITLE
Capturing scale events for ROSA

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Capturing scale events for ROSA so that we can use them in kube-burner-ocp workers-scale workload.

### Testing
Needs to be tested in a follow up PR that reads these timestamps from kube-burner-ocp code.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.18-nightly-x86-control-plane-3nodes

**Comment by paigerube14** [comment]
Is the point here to be able to see the timings or are you going to use later in other scripts? if the first, I don't see the file here. Think you'll want to use ARTIFACT_DIR instead of SHARED_DIR

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/58004/rehearse-58004-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.18-nightly-x86-control-plane-3nodes/1848171416524951552/artifacts/control-plane-3nodes/rosa-cluster-wait-ready-nodes/

File: ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh [hunk 1]
```diff
@@ -197,6 +198,8 @@ function fixNodeScaling {
     else
       rosa edit machinepool -c "$CLUSTER_ID" worker --enable-autoscaling=false --replicas "$REPLICAS"
     fi
+    workers_scale_event_epoch=$(($(date +%s) - 600))
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh]: Its overlooked earlier. Updated now.

**Comment by vishnuchalla** [comment]
> Missed your comment in the PR, might be helpful to also copy to the artifact dir to see timings. Think you'll want to use ARTIFACT_DIR
> 
> https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/58004/rehearse-58004-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.18-nightly-x86-control-plane-3nodes/1848171416524951552/artifacts/control-plane-3nodes/rosa-cluster-wait-ready-nodes/

We do not want these timestamps in ARTIFACT_DIR at the moment. We only want them in a shared directory.

**Pull request merged**